contracts/token/ERC20/extensions/IERC20Permit.sol

```diff
@@ -10,6 +10,26 @@ pragma solidity ^0.8.20;
  * Adds the {permit} method, which can be used to change an account's ERC20 allowance (see {IERC20-allowance}) by
  * presenting a message signed by the account. By not relying on {IERC20-approve}, the token holder account doesn't
  * need to send a transaction, and thus is not required to hold Ether at all.
+ *
+ * ==== Security Considerations
```

Comment:
@Amxx I don't think the utilities page was the right location for these docs. I moved it here and made some changes to it.